src/palm-core/libs/navigation.ts

```diff
@@ -38,7 +36,7 @@ export enum Routes {
   HomeTabs = 'HomeTabs',
   Explore = 'Explore',
   LensFriends = 'LensFriends',
-  NftList = 'NftList',
+  NftSelect = 'NftSelect',
```

Comment:
is there a way to preserver git changes? File renaming seems to consider this as removing the previous file and creating a new one. Hard to compare changes. hmm..

Reply:
It was an unused name. I changed it because I needed a screen to select an nft of a different design.